internal/cmd/resourcemanager.go

```diff
@@ -97,11 +104,11 @@ func runResourceTop(options *resourceTopOptions) func(cmd *cobra.Command, args [
 		var labeler envLabeler
 		switch options.group {
 		case "user":
-			groups, labeler = aggregateByUser(providers.Kubernetes, users, orgs, envs, *options)
+			groups, labeler = aggregateByUser(providers.Kubernetes, users, orgs, envs, images, *options)
 		case "org":
-			groups, labeler = aggregateByOrg(providers.Kubernetes, users, orgs, envs, *options)
+			groups, labeler = aggregateByOrg(providers.Kubernetes, users, orgs, envs, images, *options)
 		case "provider":
-			groups, labeler = aggregateByProvider(providers.Kubernetes, users, orgs, envs, *options)
+			groups, labeler = aggregateByProvider(providers.Kubernetes, users, orgs, envs, images, *options)
```

Comment:
Would defining and passing some sort of `aggregationCfg` struct for collecting these params be less redundant?

Reply:
yeah true, the number of args has gotten pretty ridiculous 😄